tests/wpt/include.ini

```diff
@@ -19,20 +19,34 @@ skip: true
     skip: false
   [CSS2]
     skip: false
+    [linebox]
+      skip:false
```

Comment:
Possibly the missing space here?

Reply:
Looks like I had an extra level of indentation in the `css-align` and `css-color` sections. Should be fixed now.